Utilities/CodeGeneration/EdFi.Ods.CodeGen/Generators/EntityOrmMappingsForQueries.cs

```diff
@@ -10,7 +10,7 @@ namespace EdFi.Ods.CodeGen.Generators
 {
     public class EntityOrmMappingsForQueries : EntityOrmMappings
     {
-        public EntityOrmMappingsForQueries(IViewsProvider viewsProvider, IDatabaseTypeTranslator databaseTypeTranslator)
+        public EntityOrmMappingsForQueries(IAuthorizationDatabaseTableViewsProvider viewsProvider, IDatabaseTypeTranslator databaseTypeTranslator)
```

Comment:
Same here, rename the field and constructor param.

Reply:
I fixed it .Thanks